### ISSUE TITLE
Issues while building SUSE directory.

### ISSUE THREAD

**Issue by GautamSatish** [OpenSCAP/scap-security-guide#1469]
Hi folks,

I am trying a "make" on the SUSE directory and facing multiple issues.

~~1) There is an extra quote character that breaks guide.xslt.~~
2) Makefile refers to disa-os-srg-v1r1.xml which is not present.
~~3) combineremediations.py and combineovals.py cannot parse platforms with 2 digit versions.~~
4) The platform string does not match the string in the map_product function.
5) The CPE check refers to installed_OS_is_sle11 which is not present.
~~6) The CPE id is not the same as the one in openSCAP tool.~~

Currently I am looking at 5 and 6, there is a chance that there might be more errors uncovered after this.

The CPE version in SSG : 
`"cpe:/o:suse:linux_enterprise_server:11" for "SUSE Linux Enterprise Server 11"`

The CPEs in openSCAP : 
```
"cpe:/o:suse:sle" for "SUSE Linux Enterprise all versions"
"cpe:/o:suse:sles:11" for "SUSE Linux Enterprise Server 11"
```

Should I rename the SSG CPE to match openSCAP and use the OVAL implementation?

Thank you.

Regards,
Gautam.

**Comment by redhatrises** [comment]
Item number 6 is invalid. The CPE version in SSG is correct; however, it is incorrect in OpenSCAP.

**Comment by Joeri-MS** [comment]
On an fresh installed SLES 12 SP1 server the /etc/os-release looks like this:
```
NAME="SLES"
VERSION="12-SP1"
VERSION_ID="12.1"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP1"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp1"
```

**Comment by redhatrises** [comment]
OpenSCAP and SSG don't read `/etc/os-release` to get the CPE. `cpe:/o:suse:sles:12:sp1` is also not a valid NIST CPE which is interesting as SUSE goes through NIST certification. See https://web.nvd.nist.gov/view/cpe/search/results?keyword=suse&status=FINAL&orderBy=CPEURI&namingFormat=2.3&startIndex=0 for valid SUSE CPEs.

**Comment by radzy** [comment]
@Joeri-MS :
OpenSCAP does check for the presence of /etc/*-release at one point, but it is a minor test that does not have anything significant to do with generating the CPE.

I've been toying with the thought of modifying it to check the contents of /etc/os-release specifically, instead of relying on the existence of /etc/wrlinux-release.  That would be more in line with Wind River's normal preferences, and it looks like it might be useful for SUSE as well.   But it's still relatively minor and not used for CPE generation during XCCDF evaluation.

**Comment by GautamSatish** [comment]
This issue is resolved by the changes made in PRs #1492, #1501 and #1506.
